src/job/MomoJobDescription.ts

```diff
@@ -0,0 +1,19 @@
+import { JobEntity } from '../repository/JobEntity';
+
+export interface SchedulerStatus {
+  interval: string;
+  started: boolean;
+}
+
+export interface MomoJobDescription {
```

Comment:
Can we document the new types a bit? Especially what it means to have/not to have a `schedulerStatus`.

Reply:
I added a section to the readme

Reply:
Nice! Could we also add some inline docs so the IDE explains it a bit?